### DISCUSSION TITLE
Minimum requirements for selfhosting just the application

### DISCUSSION THREAD

**Discussion by proton029** [Budibase/budibase#7290] [Help]
I have the budibase application selfhosted with the application i built published in it. I want to know whether i can remove some of the docker images so that i can reduce the resource consumption.

**Comment by melohagan** [comment]
Hey @proton029 

Each of the containers play an important role in the functioning of your applications. 

If you're having resource issues, the single image container should be a little less resource intensive: https://docs.budibase.com/docs/docker#method-1---docker-run
